internal/collector/job_test.go

```diff
@@ -42,11 +42,15 @@ var (
 )
 
 func TestJobCollector(t *testing.T) {
+	var test = true
```

Comment:
Tiny thing: I would prefer:

```golang
var trueValue = true
```

What do you think?

Reply:
Taken straight from the `pod_test.go`, but I'll change it.